whats_new/new-features.adoc

```diff
@@ -60,7 +60,7 @@ Easy, over-the-air upgrades for asynchronous z-stream releases of
 OpenShift v4 is available. Cluster administrators can upgrade using the
 *Cluster Settings* tab in the web console.
 See
-xref:../updating/updating-cluster-within-minor.adoc#updating-cluster-within-minor[Updating a cluster]
+xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-within-minor[Updating a cluster]
```

Comment:
Same `#`-fragment considerations discussed [here][1].

[1]: https://github.com/openshift/openshift-docs/pull/61201#discussion_r1230209175

Reply:
Hmm and also, while this was probably a reasonable link target back when this section was written, it may be worth updating it to drop the web-console specificity and link to the updates section as a whole.  Possible with wording like "using the web console or the OpenShift CLI (`oc`)" that we currently use in `modules/update-conditional-updates.adoc`.  No preference about whether that sort of thing happens in this  pull or in follow-up work.

Reply:
I'll save this for a follow up